Word_Counter/script.js

```diff
@@ -0,0 +1,33 @@
+function setInput() {
+  var input = document.getElementById("input");
+
+  function isWord(string) {
+    var isAlphaNumeric = false;
```

Comment:
![image](https://user-images.githubusercontent.com/66599363/135307546-dc430530-e36a-4eac-8f4d-495c589f7760.png)
Make the text area wider as I write into it count option is gone? I Can't see the how many words is count fix this.

Reply:
okay. I'll fix it. Thanks

Reply:
@soumik2012  Changes resolved. Please review it. Thanks